docs(data-warehouse): Add anchor time documentation with timezone behavior (#18580)
PostHog/posthog.com: contents/docs/cdp/sources/index.mdx

CDC (Change Data Capture) schemas support sync frequencies as low as every 1 minute. All other sync methods (Incremental, Full table, Append only) have a minimum sync frequency of 5 minutes. This restriction is enforced in both the UI and the API.

### Anchor time

Anchor time pins the sync schedule so runs start at a predictable time each day. This is useful for coordinating syncs with downstream jobs or reports. Anchor time only applies to sync intervals longer than one hour.

> The anchor time is stored in UTC. When your project timezone is set to UTC or GMT, the UI displays "(UTC)" next to the anchor time label. For other project timezones, a toggle lets you view and set the anchor time in either UTC or your project timezone.

Reviewer: ```suggestion
The anchor time is stored in UTC. When your project timezone is something other than UTC, a toggle lets you view and set the anchor time in either UTC or your project timezone.
```
